modules/ROOT/pages/provisioning-vmware.adoc

```diff
@@ -155,3 +156,31 @@ VM_NAME='fcos-node01'
 govc device.serial.add -vm "${VM_NAME}"
 govc device.serial.connect -vm "${VM_NAME}" "[datastore] ${VM_NAME}/console.log"
 ----
+
+== Modifying OVA metadata
```

Comment:
Let's try to distinguish OVA and OVF throughout this section.  The metadata is OVF and the single-file archive is OVA.